File: 2022-2023/Kurs_programowania/Lista 3/Zadanie 2/Kwadrat.cpp
#include "Kwadrat.hpp"
Kwadrat::Kwadrat(double bok1, double bok2, double bok3, double bok4, double kat): Czworokat(bok1,bok2,bok3,bok4,kat) {};

double Kwadrat::obwod() {
    return bok1*4;
}
double Kwadrat::pole() {
    return bok1*bok1;
}

std::string Kwadrat::nazwa() {
    return "Kwadrat";
}

File: 2022-2023/Kurs_programowania/Lista 3/Zadanie 1/Kwadrat.java
public class Kwadrat extends Czworokat{
    public Kwadrat(double bok){
        super("kwadrat",bok,bok,bok,bok);
    }
    public double obliczPole(){
        return bok1*bok1;
    }
    public double obliczObwod(){
        return 4*bok1;
    }
}
